### PR TITLE
Modify order total before creation

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: features/modifying_order_total.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@managing_orders
+Feature: Modifying order total
```
Comment (bartoszpietrzak1994) [inline features/modifying_order_total.feature]: What about custom decimal amount and testing decimals containing three or more decimal places?

File: src/Entity/Order.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Sylius\AdminOrderCreationPlugin\Entity;
+
+use Sylius\Component\Core\Model\Order as BaseOrder;
+
+class Order extends BaseOrder
+{
+    /** @var int|null */
```
[the remark this answers is not in the document]
Comment (Zales0123) [reply src/Entity/Order.php]: Prices in Sylius are kept as integers (cents not euros, for example). I agree that it should be VO but want to be consistent with current Sylius approach.

File: features/modifying_order_total.feature (hunk 1 above)
Comment (Zales0123) [reply features/modifying_order_total.feature]: We can test decimal totals, for sure, but about "three or more decimal places" - I don't know is it crucial for now, it does not provide any valid value imo

Comment (bartoszpietrzak1994) [reply features/modifying_order_total.feature]: ok then

File: src/Entity/Order.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (bartoszpietrzak1994) [reply src/Entity/Order.php]: ok then

**Pull request merged**